pkg/sql/logictest/testdata/logic_test/show_source

```diff
@@ -270,108 +270,112 @@ SELECT schema_name, table_name, type, owner, locality FROM [SHOW TABLES FROM sys
 ORDER BY schema_name, table_name
 ----
 schema_name  table_name                       type      owner  locality
+public       comments                         table     NULL   NULL
```

Comment:
hrm, it's weird that the out put was not ordered by table name and the test was passing?